app/views/podcast_planner/_form_options.html.erb

```diff
@@ -0,0 +1,33 @@
+<section>
```

Comment:
these particular options have no impact on date generation so i separated them out into their own partial for conceptual clarity. i even wonder if they can just go into the side panel and just be selected in the preview?